Dockerfile

```diff
@@ -1,5 +1,24 @@
-FROM ruby:2.5.1
-RUN apt-get update -qq && apt-get install -y ruby-dev
+FROM alpine:3.8
+
+LABEL org.label-schema.maintainer="Puppet Bolt Team <team-direct-change-bolt@puppet.com>" \
+      org.label-schema.vendor="Puppet" \
+      org.label-schema.url="https://github.com/puppetlabs/bolt" \
+      org.label-schema.name="PE Bolt Server" \
+      org.label-schema.license="Apache-2.0" \
+      # Is there a way to set this?
+      #org.label-schema.version=$PUPPET_VERSION \
```

Comment:
Solution to the version? I'm not sure what hooks to set up to have it auto-update when we update the version.

Reply:
I'm not sure. Probably worth asking a wider audience.

Reply:
Updating it may just have to be part of the tag-and-release job.

Reply:
It seems like this only matters for the containers we publish.
Can we use an ARG and pass it when we build the container for publishing? Then we can ensure we use the same version here as in the tag.

Reply:
This is what I'm doing
```yaml 
script:
    - export BOLT_VERSION=`git describe`
    - export BOLT_TAG=$REGISTRY_HOST/$DOCKER_REGISTRY/bolt-server:$BOLT_VERSION
    - docker login -u TOKEN -p $DOCKER_TOKEN $REGISTRY_HOST
    - docker build --tag $BOLT_TAG --build-arg bolt_version=$BOLT_VERSION ./
    - docker push $BOLT_TAG
```
so it should work if you add

```Dockerfile
ARG bolt_version=no-version
org.label-schema.version=${bolt_version}
````